### PR TITLE
feat: serial workflow advance fix

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Check both `graded` and `complete` workflow status indicators to confirm we've completed a workflow step instead of just `complete`.

**What changed?**

In the new ORA UI we serially progress through steps, blocking movement forward to a subsequent step until all requirements are met for the current step. This blocks us from determining workflow status directly (as previously, we would jump forward as far as you are able to view, skipping peer if it was followed by an additional step) and requires us to manually determine how far we have progressed.

Previous checks incorrectly determined this by only checking `complete` (whether *you* have finished your required work on a step) instead of also checking `graded` (whether you have received required external grades e.g. from peers).

This adds the additional check that both `complete` and `graded` must be `True` for a step to be considered finished.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by mattcarter — APPROVED** [review]
Approved.

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍
